firefox-ios/firefox-ios-tests/Tests/ClientTests/ContentBlockerTests.swift

```diff
@@ -21,9 +21,12 @@ final class ContentBlockerTests: XCTestCase {
 
     func testCompileListsNotInStore_callsCompletionHandlerSuccessfully() {
         let expectation = XCTestExpectation()
+        let startDate = Date()
         ContentBlocker.shared.compileListsNotInStore {
+            let finishDate = Date()
+            print(finishDate.timeIntervalSince(startDate))
```

Comment:
can you put some pointers like "MT Test Time - \(finishDate.timeIntervalSince(startDate))" to make it easier to find it, also this won't be pushed to production so we can play around with adding nslog as well

Reply:
Will add better debug logging 😅 

Also, since the test failure is intermittent, would it be worth merging to production, that way we can passively gather these metrics with all other PR's instead of just re-running this one until it occurs again?

Reply:
Sorry fo the late reply here. Yeah, we try not to use print statments in our tests.

In the [Performance tests](https://github.com/mozilla-mobile/firefox-ios/blob/6a3bf6db3949c19ed14946765dfd955933b412cf/firefox-ios/firefox-ios-tests/Tests/XCUITests/PerformanceTests.swift#L7) we use [measure](https://developer.apple.com/documentation/xctest/xctestcase/3194265-measure) and the metrics its provides. Not sure if that would serve your needs... but just in case